### PR TITLE
Fix warnings and deprecations

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: src/main/java/com/redhat/cloud/policies/app/auth/IncomingRequestFilter.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
         producer.setPrincipal(rhPrincipal);
     }
 
-    private XRhIdentity determineXRhIdentity(ContainerRequestContext requestContext, String xrhid_header) {
+    private XRhIdentity determineXRhIdentity(String xrhid_header) {
```
Comment (bennetelli) [inline src/main/java/com/redhat/cloud/policies/app/auth/IncomingRequestFilter.java:110]: unused parameter

File: src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
  */
 package com.redhat.cloud.policies.app;
 
+import static java.util.Calendar.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java:19]: Star import

**Review by pilhuhn — APPROVED** [review]
Cosmetics, otherwise LGTM

**Comment by bennetelli** [comment]
Forgot to push while preparing lunch.

**Pull request merged**